WcaOnRails/lib/results_validators/persons_validator.rb

```diff
@@ -111,6 +114,25 @@ def validate(competition_ids: [], model: Result, results: nil)
           if without_wca_id.select { |p2| p2.name == p.name }.length > 1 && !duplicate_newcomer_names.include?(p.name)
             duplicate_newcomer_names << p.name
           end
+          # Look for obvious person name issues
+          splitted_name = p.name.split
```

Comment:
Nit on spelling. I would probably calling this `splt_name`.

"Splitted" is [an archaic past tense spelling](https://www.merriam-webster.com/dictionary/splitted) and doesn't read well.